openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -214,6 +224,58 @@ This is an example of a Wallet Instance Attestation:
     * In the `input-descriptors` object, `id`, `name`, `purpose`, `group`, `format`, and `constraints` properties MUST be supported. In the `constraints` object, `limit_disclosure`, and `fields` properties MUST be supported. In the `fields` object, `path` and `filter` properties MUST be supported. A `path` MUST contain exactly one entry with a static path to a certain claim. A `filter` MUST only contain `type` elements of value `string` and `const` elements.
     * In the `submission_requirements` object, `name`, `rule (`pick` only)`, `count`, `from` properties MUST be supported.
 
+# OpenID for Verifiable Presentations over W3C Digital Credentials API
+
+The requirements for the Wallet and the Verifier, unless specified otherwise:
+
+* MUST support Annex A in [@!OIDF.OID4VP] that defines how to use OID4VP over the W3C Digital Credentials API.
+  * The Wallet MUST support both signed and unsigned requests defined in Annex A.3.1 and A.3.2 of [@!OIDF.OID4VP]. The Verifier MUST support signed requests, unsigned requests, or both.
+* Wallet Invocation is done via the W3C Digital Credentials API or an equivalent platform API. Custom URL schemes MUST NOT be used.
+* Response Mode MUST be `dc_api.jwt`. The response MUST be encrypted.
+* The DQCL query and response as defined in Section 6 of [@!OIDF.OID4VP] MUST be used. Presentation Exchange as defined in Sections 5.4 and 5.5 of [@!OIDF.OID4VP] MUST NOT be used. Below is the list of features in the DQCL query and response that MUST be supported:
+  * tbd
+* Support for Transaction Data as defined in Sections 5.4 and 7.4 of [@!OIDF.OID4VP] is tbd.
+
+## ISO mdoc specific requirements for OpenID for Verifiable Presentations over W3C Digital Credentials API
```

Comment:
Because Annex B.3 basically says "see 18013 part 7", and whilst we have agreement with ISO to look at doing something different (incorporating non-DC API mdoc into HAIP), for now we don't want to introduce that - and part 7 doesn't use DCQL and depends on an old version of OID4VP that doesn't support the DC API.

Reply:
because this section defines session transcript etc. that cannot be normatively defined in oid4vp. + as Joseph said, OID4VP points to 18013-7 and there is huge resistance to point to 18013-7, because it is behind the paywall